src/layouts/BlogPostLayout.astro

```diff
@@ -9,10 +10,13 @@ const {frontmatter} = Astro.props;
   <head>
     <meta charset="utf-8">
   <!-- <link rel="stylesheet" type="text/css" href="/src/styles/styles.css"> -->
+  <link href="/dist/css/decanter.css" rel="stylesheet">
 	</head>
-  <h1>{frontmatter.title}</h1>
-  <h2>Post author: {frontmatter.author}</h2>
-  <p>{frontmatter.description}</p>
-  <slot /> <!-- Markdown content is injected here -->
-   <!-- ... -->
+  <Container width="site">
+    <h1 class="text-cardinal-red">{frontmatter.title}</h1>
+    <h2 class="text-18">Post author: {frontmatter.author}</h2>
```

Comment:
I included some decanter v7 utility classes here as an example (and to confirm that it's working) 😆